hygiene_string.js

```diff
@@ -0,0 +1,12 @@
+conn = new Mongo();
+db = conn.getDB("weisheng");
+
+db.hygiene.find({'PHONE':{$type:18}}).forEach(
+	function(e) {db.hygiene.update({_id:e._id}, {$set:{PHONE: ""+e.PHONE}})}
+	);
+db.hygiene.find({'PHONE':{$type:16}}).forEach(
+	function(e) {db.hygiene.update({_id:e._id}, {$set:{PHONE: e.PHONE.toString()}})}
+	);
+db.hygiene.find({'PHONE':{$exists:true}}).forEach(
+	function(e) {db.hygiene.update({_id:e._id}, {$set:{PHONE: "+1"+e.PHONE}})}
+	);
```

Comment:
Yes, you are right. This code is just for formatting the phone numbers. And the code for join is easy, but I'm not sure the result is proper or not, So we can do the formatting and join separately. I think next meet we can try the join and see whether the result is correct or not.